deis/charts
# feat(monitor): Allow for off cluster influxdb

Test steps:
- clone PR
- In generate_params.toml
  - Mark the property `influxdb_location` as `off-cluster`
  - Add values under influxdb for url, user, password, and database
- run `helm generate`
- verify that configuration values were inserted into `manifests/deis-monitor-telegraf-daemon.yaml` and `manifests/deis-monitor-grafana-deployment.yaml`
